alert-common/src/main/java/com/synopsys/integration/alert/common/descriptor/config/ui/ProviderDistributionUIConfig.java

```diff
@@ -117,9 +116,23 @@ private LabelValueSelectOption convertToLabelValueOption(ProviderNotificationTyp
         return new LabelValueSelectOption(notificationTypeLabel, notificationType);
     }
 
-    private LabelValueSelectOption convertToLabelValueOption(String formatName) {
-        String formatNameLabel = WordUtils.capitalizeFully(formatName.replace("_", " "));
-        return new LabelValueSelectOption(formatNameLabel, formatName);
+    private LabelValueSelectOption convertToLabelValueOption(FormatType formatType) {
+        return new LabelValueSelectOption(formatType.getLabel(), formatType.name());
+    }
+
+    private String createFormatDescription() {
+        StringBuilder formatDescription = new StringBuilder();
+        for (FormatType format : providerContent.getSupportedContentFormats()) {
+            String label = format.getLabel();
+            String description = format.getDescription();
+
+            formatDescription.append(label);
+            formatDescription.append(": ");
```

Comment:
Isn't the ':' already there? The DESCRIPTION_FORMAT now ends in ':'

Reply:
This colon is located after the format name whereas the other colon is after the generic format description before we describe each one.